### DISCUSSION TITLE
0% feed-in with schedule

### DISCUSSION THREAD

**Discussion by Cartman586** [community/community#79265] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

A 0% feed-in with a schedule would be good.
Example
20.00-00.00 100% power
00.00-06.00 0% feed-in

Flexibly adjustable

**Comment by 1lyosbek** [comment]
yes

**Comment by 1lyosbek** [comment]
ha

**Comment by 1lyosbek** [comment]
ha alnbatta]
